src/main/java/io/fusionauth/domain/UsageDataConfiguration.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2024-2024, FusionAuth, All Rights Reserved
```

Comment:
This would be in `fusionauth-app` - but it looks like your copyright got messed up.

Reply:
Fixed.